### PR TITLE
feat: import path overrides

### PR DESCRIPTION
**Pull request by nklaassen**
This commit adds a configuration option to override the import path for specific Go packages, in case they are not correctly found by the plugin.

My usecase is generating schemas for a type defined outside of the github.com/gravitational/teleport/api/types package. Without this change the plugin was just trying the following, which obviously doesn't work:

```go
import (
	types    "types"
	wrappers "wrappers"
)
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,14 +49,15 @@ Please also specify the full name of the go package where your generated code is
 default_package_name="github.com/gravitational/teleport/api/types"
 ```
 
-If you use types from external packages, please specify them in `external_imports`:
```
Comment (nklaassen) [inline README.md:52]: Note: I could not find any reference to this `external_imports` field or any way of using it

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-v1.4.0
\ No newline at end of file
+v1.4.1
```
Comment (nklaassen) [inline VERSION:1]: I will create this git tag after merging

File: README.md (hunk 1 above)
Comment (marcoandredinis) [reply README.md:52]: Yeah, not sure where that comes from.
Possibly something that didn't end up being implemented :shrug:

File: VERSION (hunk 2 above)
Comment (marcoandredinis) [reply VERSION:1]: There's no need to tag
Just ensure you add the version constraint in the `teleport-plugins` repo when running `make terraform-gen-tfschema`

**Review by marcoandredinis — APPROVED** [review]
LGTM

**Review by r0mant — APPROVED** [review]

**Pull request merged**